<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ikoOSKAR::UI::AuthenticatorWindow</class>
 <widget class="QMainWindow" name="ikoOSKAR::UI::AuthenticatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ikoOSKAR'ı Etkinleştirin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icon-WhiteBGx128.png</normaloff>:/Icon-WhiteBGx128.png</iconset>
  </property>
  <widget class="QWidget" name="root">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="header">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblBanner">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>82</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="content" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QGroupBox" name="grpLicense">
         <property name="title">
          <string>Lisanslı Kullanıcı</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>24</number>
          </property>
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ikoOSKAR'a hoş geldiniz. Ürünü etkinleştirmek için lütfen lisans anahtarınızı giriniz. Bilgisayarınızın internete bağlı olduğundan emin olunuz.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Lisans anahtarınızı giriniz: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtSerial">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnActivateLicense">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Etkinleştir</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpDemo">
         <property name="title">
          <string>Yeni Kullanıcı</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Eğer lisans anahtarınız yoksa ücretsiz deneme sürümünü başlatabilir ya da aşağıdaki e-posta adresinden iletişime geçerek bir lisans anahtarı satın alabilirsiniz.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>87</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnStartDemo">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ücretsiz Deneme Sürümünü Başlat</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>86</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpContact">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>İletişim</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>328</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="contactCards" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>30</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>327</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtSerial</sender>
   <signal>returnPressed()</signal>
   <receiver>btnActivateLicense</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
